build.gradle.kts

```diff
@@ -2,4 +2,5 @@
 plugins {
     alias(libs.plugins.androidApplication) apply false
     alias(libs.plugins.jetbrainsKotlinAndroid) apply false
+    id("com.google.gms.google-services") version "4.4.0" apply false
```

Comment:
It is okay like that but I didn't do exactly the same in my PR we can discuss about that when we see each other

Reply:
Ok we will see!